scripts/jenkins-yoctobuild-build.sh

```diff
@@ -177,5 +185,32 @@ if [ "$RUN_INTEGRATION_TESTS" = "true" ]; then
     if [ $PUBLISH_ARTIFACTS = "true" ]; then
         s3cmd -F put core-image-full-cmdline-vexpress-qemu.ext4 s3://mender/temp/core-image-full-cmdline-vexpress-qemu.ext4
         s3cmd setacl s3://mender/temp/core-image-full-cmdline-vexpress-qemu.ext4 --acl-public
+
+        if [ ! -z ${RELEASE_VERSION} ]; then
```

Comment:
Oops sorry, I missed this the first time around. Since you are not using `[[`, I think this test requires quotes around `$RELEASE_VERSION` in order to be safe (without quotes it may expand to a non-argument, which will produce a syntax error rather than a failed expression).